src/renderer/js/redux/actions/subscriptions.js

```diff
@@ -0,0 +1,21 @@
+import * as actions from "constants/action_types";
```

Comment:
no flow on this page?

Reply:
whoops. I added the types, but just forgot to re-add the flow comment